examples/flutter_gallery/lib/demo/material/data_table_demo.dart

```diff
@@ -77,6 +78,7 @@ class DessertDataSource extends DataTableSource {
     Dessert('Coconut slice and honeycomb',          567, 18.2,  94,  8.0, 571,  0, 51),
     Dessert('Coconut slice and donut',              611, 40.0,  58,  6.4, 335,  2, 28),
     Dessert('Coconut slice and KitKat',             677, 41.0,  72,  8.5,  63, 12, 12),
+    // @formatter:on
```

Comment:
i'm not a huge fan of having to have pragmas like this in the code

ideally the formatter would actually format this code like this so we wouldn't have to do it manually

Reply:
Agree but really to hard in short term.